site/en/examples/train_text_classifier_embeddings.ipynb

```diff
@@ -653,7 +653,7 @@
         "\n",
```

Comment:
<p>Is it worth updating this embed call as well? Looks like <code>tqdm</code> can work with <code>df.apply</code> too. </p><p> </p><p>e.g.</p><pre class="ql-syntax" spellcheck="false">from tqdm.auto import tqdm
tqdm.pandas()

from google.api_core import retry

def make_embed_text_fn(model):

  @retry.Retry(timeout=300.0)
  def embed_fn(text: str) -&gt; list[float]:
    return palm.generate_embeddings(model=model, text=text)['embedding']

  return embed_fn

def create_embeddings(model, df):
  df['Embeddings'] = df['Text'].progress_apply(make_embed_text_fn(model))
</pre><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/generative-ai-docs/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>